tests/vmi_hook_sidecar_test.go

```diff
@@ -303,10 +303,22 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 
 		Context("with ConfigMap in sidecar hook annotation", func() {
 
-			It("should update domain XML with SM BIOS properties", func() {
+			It("should update domain XML with SM BIOS properties when sidecar image is specified", func() {
 				cm, err := virtClient.CoreV1().ConfigMaps(testsuite.GetTestNamespace(vmi)).Create(context.TODO(), RenderConfigMap(), metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				vmi.ObjectMeta.Annotations = RenderSidecarWithConfigMap(hooksv1alpha2.Version, cm.Name)
+				vmi.ObjectMeta.Annotations = RenderSidecarWithConfigMapPlusImage(hooksv1alpha2.Version, cm.Name)
+				vmi = tests.RunVMIAndExpectLaunch(vmi, 360)
+				domainXml, err := tests.GetRunningVirtualMachineInstanceDomainXML(virtClient, vmi)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(domainXml).Should(ContainSubstring("<sysinfo type='smbios'>"))
+				Expect(domainXml).Should(ContainSubstring("<smbios mode='sysinfo'/>"))
+				Expect(domainXml).Should(ContainSubstring("<entry name='manufacturer'>Radical Edward</entry>"))
+			})
+
+			It("should update domain XML with SM BIOS properties when sidecar image is not specified", func() {
```

Comment:
The two tests look almost identical. Could we parametrize them as needed and use `DescribeTable` to avoid duplicated code?

Reply:
Done.